src/components/nav-bar/nav-bar.tsx

```diff
@@ -25,59 +28,84 @@ export const NavBar = () => {
       key: "settings",
       icon: <SettingOutlined />,
       label: t("settings"),
+      onClick: () => {
+        console.log("settings clicked");
+        setIsSettingSelected(true);
+      },
     },
     {
       key: "logout",
       icon: <LogoutOutlined />,
-      label:t("logout"),
+      label: t("logout"),
       className: "menu-logout",
       onClick: () =>
         logout({ logoutParams: { returnTo: window.location.origin } }),
     },
   ];
 
   return (
-    <div className="navbar">
-      <div className="left-nav">
-        <img src="/src/assets/oz-mern.png" alt="Logo" className="logo" />
-        <span className="app-name">{t("app-name")}</span>
-      </div>
-      <ThemeSwitcher />
-      <TranslationSwitcher/>
-      <div className="right-nav">
-        <div className="connection-wrapper">
-          <ConnectionHealth />
+    <>
+      <div className="navbar">
+        <div className="left-nav">
+          <img src="/src/assets/oz-mern.png" alt="Logo" className="logo" />
+          <span className="app-name">{t("app-name")}</span>
         </div>
-        <div className="auth-container">
-          {isAuthenticated ? (
-            <>
-              <div className="user-photo">
-                <Avatar
-                  src={user?.picture}
-                  icon={<UserOutlined />}
-                  alt="User"
-                />
-              </div>
-              <div className="user-name">{user?.name || "Unknown"}</div>
-              <div className="user-menu">
+        <ThemeSwitcher />
```

Comment:
I think Lines 53 - 54 can be removed, you placed the settings in the modal

Reply:
> I think Lines 53 - 54 can be removed, you placed the settings in the modal

I keep them to let you or Ofir to choose which way is better (setting modal or nav-bar switchers).